### DISCUSSION TITLE
Call to an undefined method App\Actions\Common\BaseQueryBuilder::allowedAppends()

### DISCUSSION THREAD

**Discussion by imfaisii** [spatie/laravel-query-builder#879] [Q&A]
I am using a Query Trait to allowAppends.
On the latest version of Spatie I get this error: `Call to an undefined method App\Actions\Common\BaseQueryBuilder::allowedAppends()`

`QueryBuilder::for($query)
            ->allowedFilters(array_merge($model->getAllowedFilters(), $this->getExtraFilters()))
            ->allowedAppends(array_merge($model->getAllowedAppends(), $this->getExtraAppends()))`

**Comment by ludmanp** [comment]
@imfaisii Did you resolve the issue?

**Reply by ghost** [reply]
Same here on v5, it used to work on v4
